SDKKeyStore.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+
+@objc public class SDKKeyStore: NSObject {
```

Comment:
Can we just call it `KeyStore` and put it inside a `KeyStore` folder like was done for `StandardHeaders`?

Reply:
I've updated this.